src/components/panel/annotations/AnnotationsListMode.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from 'react'
+import { usePanel } from '@/contexts/PanelContext.tsx'
+
+import Annotation from '@/components/panel/annotations/Annotation.tsx'
+
+import { getFilteredAnnotations } from '@/utils/annotations.ts'
+
+
+
+const AnnotationsListMode: FC = () => {
```

Comment:
The component is not a mode. The component is rendered when that mode becomes active, so the comp itself is acutally independent and should be considered as such. So "AnnotationsList" fits better.